lib/cartopy/tests/test_crs.py

```diff
@@ -11,8 +11,9 @@
 import pickle
 
 import numpy as np
-from numpy.testing import assert_almost_equal, assert_array_equal
+from numpy.testing import assert_almost_equal
 from numpy.testing import assert_array_almost_equal as assert_arr_almost_eq
+from numpy.testing import assert_array_equal
```

Comment:
Did it really split these? I thought it would just re-order.

Reply:
Hmmm I'll see if I can reach a compromise here...

Reply:
[combine_as_imports](https://pycqa.github.io/isort/docs/configuration/options.html#combine-as-imports) is our friend here :+1:

Reply:
@QuLogic Done, see 346473c54d36ab0af149fb666179b8b367948354

Reply:
Ah, I wasn't asking about the `as` line here, but why `assert_almost_equal` and `assert_array_equal` were split up. But I think I understand it now with that context. It sorted the imported symbols, and since the `as` line was _between_ the two others (and `combine_as_imports` was off), they became split.

I wasn't particularly against this change; just curious why it happened. I think either option is good.